pkg/db/v1beta1/postgres/postgres_test.go

```diff
@@ -136,4 +136,10 @@ func TestGetDbName(t *testing.T) {
 		t.Errorf("getDbName returns wrong value %v", getDbName())
 	}
 
+	sslMode := "require"
+	t.Setenv(common.PostgreSSLMode, sslMode)
+	dbName = "host=katib-postgres port=5432 user=katib password= dbname=katib sslmode=require"
+	if getDbName() != dbName {
+		t.Errorf("getDbName returns wrong value %v", getDbName())
+	}
```

Comment:
```
	cases := map[string]struct{
		updateEnvs map[string]string
		wantName string	
	}{
		"All parameters are default": {
		...
		},
		"Set KATIB_POSTGRESQL_SSL_MODE": {
		...
		}
	}
	for name, tc := range cases {
		t.Run(name, func(t *testsing.T) {
			// TEST
		})
	}
```

Please could you refine these tests?

Reply:
@tenzen-y Oh I got your point. I made the change based on your suggestion. And add more test cases for each env. Please help to review it. Thanks.

Reply:
@ckcd Thanks for your great refining! I'd be appreciated for your effort!